symengine/lib/symengine_wrapper.pyx

```diff
@@ -4447,7 +4447,7 @@ cdef class _Lambdify(object):
             raise ValueError("Size of out incompatible with number of exprs.")
         self.unsafe_complex(inp, out)
 
-    def __call__(self, inp, *, out=None):
+    def __call__(self, *args, out=None):
```

Comment:
This will stop us from adding a second positional argument in the future. I suppose that's alright since kw-only arguments work well.

Reply:
Yes, any new arguments can be made keyword only.